### PR TITLE
Fix typos

### LINKED ISSUE #233: Typo  in src/content/1.8/functoriality.tex line 30
Hi,
line 30 should be:
category $\cat{C}\times{}\cat{D}$. We define a morphism in a Cartesian product of categories

this is already fixed here:
https://bartoszmilewski.com/2015/02/03/functoriality/

### LINKED ISSUE #265: Typo in Preface
Lines 43-44:

`Some time ago the principles of structural programming
revolutionized programming`

I think the correct term is _structured programming_:

[https://en.wikipedia.org/wiki/Structured_programming](url)

### PR DESCRIPTION
**Pull request by ktgw0316**
* Replace "..." with \ldots{}
* fix #233 
* fix #265

### CODE REVIEW

File: src/content/0.0/preface.tex [hunk 1]
```diff
@@ -40,7 +40,7 @@
 definition of the category itself. And I will argue strongly that
 composition is the essence of programming. We've been composing things
 forever, long before some great engineer came up with the idea of a
-subroutine. Some time ago the principles of structural programming
+subroutine. Some time ago the principles of structured programming
```
Comment (drupol) [inline src/content/0.0/preface.tex:43]: Why this change?

File: src/content/1.8/functoriality.tex [hunk 2]
```diff
@@ -27,7 +27,7 @@ \section{Bifunctors}
 morphisms, one from $\cat{C}$ and one from $\cat{D}$, to a morphism in $\cat{E}$.
 
 Again, a pair of morphisms is just a single morphism in the product
-category $\cat{C}\times{}\cat{D}$ to $\cat{E}$. We define a morphism in a Cartesian product of categories
+category $\cat{C}\times{}\cat{D}$. We define a morphism in a Cartesian product of categories
```
Comment (drupol) [inline src/content/1.8/functoriality.tex:30]: Why this change ?

**Review by drupol — CHANGES_REQUESTED** [review]

File: src/content/0.0/preface.tex (hunk 1 above)
Comment (ktgw0316) [reply src/content/0.0/preface.tex:43]: I think the term "structured programming" is more widely used, as @bigaure pointed out in #265.

File: src/content/1.8/functoriality.tex (hunk 2 above)
Comment (ktgw0316) [reply src/content/1.8/functoriality.tex:30]: This is already fixed in the [original blog post](https://bartoszmilewski.com/2015/02/03/functoriality/). I think this is just a typo. If my understanding is correct, the single morphism doesn't go from the category (C x D) to another category (E). It is not a functor.

**Review by drupol — APPROVED** [review]
Thanks!
